python-checks/src/test/java/org/sonar/python/checks/MissingNewlineAtEndOfFileCheckTest.java

```diff
@@ -20,40 +20,28 @@
 package org.sonar.python.checks;
 
 import org.junit.Test;
-import org.sonar.python.PythonAstScanner;
-import org.sonar.squidbridge.api.SourceFile;
-import org.sonar.squidbridge.checks.CheckMessagesVerifier;
+import org.sonar.python.checks.utils.PythonCheckVerifier;
 
 public class MissingNewlineAtEndOfFileCheckTest {
 
   @Test
   public void missing_new_line() {
-    SourceFile file = PythonAstScanner.scanSingleFile("src/test/resources/checks/missingNewlineAtEndOfFile1.py", new MissingNewlineAtEndOfFileCheck());
-    CheckMessagesVerifier.verify(file.getCheckMessages())
-        .next().withMessage(String.format(MissingNewlineAtEndOfFileCheck.MESSAGE, "missingNewlineAtEndOfFile1.py"))
-        .noMore();
+    PythonCheckVerifier.verify("src/test/resources/checks/missingNewlineAtEndOfFile1.py", new MissingNewlineAtEndOfFileCheck());
```

Comment:
@pynicolas You're right, unless it is static. Anyway, not truly relevant for this very rule, since it has no state.